great_expectations/analytics/client.py

```diff
@@ -23,7 +28,8 @@ def submit(event: Event) -> None:
         groups = {
             "data_context": event.data_context_id,
         }
-        if event.organization_id:
+
+        if event.organization_id and not get_config().anonymize_events:
```

Comment:
Instead of calling `get_config()` here can you plumb it inot the event. It looks like that's what we do for `organization_id`.